examples/seismic/tti/operators.py

```diff
@@ -280,14 +341,21 @@ def kernel_staggered_3d(model, u, v, space_order):
 def ForwardOperator(model, geometry, space_order=4,
                     save=False, kernel='centered', **kwargs):
     """
-       Constructor method for the forward modelling operator in an acoustic media
-
-       :param model: :class:`Model` object containing the physical parameters
-       :param src: None ot IShot() (not currently supported properly)
-       :param data: IShot() object containing the acquisition geometry and field data
-       :param: time_order: Time discretization order
-       :param: spc_order: Space discretization order
-       """
+    Construct an forward modelling operator in an acoustic media.
+
+    Parameters
+    ----------
+    model : Model
+        Object containing the physical parameters.
+    src : AcquisitionGeometry
+        The time series data for the injected source term.
+    data : ndarray
+        IShot() object containing the acquisition geometry and field data.
```

Comment:
IShot superseded by I don't know what. What should we write here @mloubout

Reply:
?

Reply:
@mloubout reminder

Reply:
@mloubout reminder